README.md

```diff
@@ -200,7 +215,7 @@ We can then request volumes from this StorageClass using the Hype API:
 
 ```scala
 import sys.process._
-import com.spotify.hype._
+import com.spotify.hype.magic._
```

Comment:
The YAML example at the bottom of the README should be updated to not contain an image. Also add a text explaining where the image comes from.